packages/flutter_tools/lib/src/base/dds.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:dds/dds.dart' as dds;
+
+import '../globals.dart' as globals;
+import 'io.dart' as io;
+
+class DartDevelopmentService {
```

Comment:
For new code, we're trying to avoid using anything in globals. This can be restructured easily enough as:

```
class DartDevelopmentService {
  DartDevelopmentService({
    this.logger,
    this.launcher = defaultValue
  });

  final Logger logger;
  final Future<dds.DartDevelopmentService> launcher;
}

```

Nit: This class doesn't do a ton but I would add a short doc comment on the purpose.